_benchmark/user-guide/understanding-workloads/common-operations.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: default
+title: Common operations
+nav_order: 16
+grand_parent: User guide
+parent: Understanding workloads
+---
+
+# Common operations
+
+[Test procedures]({{site.url}}{{site.baseurl}}/benchmark/user-guide/understanding-workloads/anatomy-of-a-workload#_operations-and-_test-procedures) use a variety of operations, found inside the `operations` directory of a workload. This page details the most common operations found inside Benchmark workloads.
```

Comment:
Should "OpenSearch" precede "Benchmark to be consistent naming convention throughout this doc (for example, line 123?)

Reply:
Yes.